<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>扫雷专业版</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/logo.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblFlag">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>标记: 2</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblTime">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>时间: 23s</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStartAction">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionStartGame"/>
    <addaction name="actionExitGame"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>等级</string>
    </property>
    <addaction name="actionLowLevel"/>
    <addaction name="actionMidLevel"/>
    <addaction name="actionHighLevel"/>
    <addaction name="separator"/>
    <addaction name="actionCustomLevel"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuStartAction"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStartGame">
   <property name="text">
    <string>开始游戏</string>
   </property>
   <property name="toolTip">
    <string>开始游戏</string>
   </property>
  </action>
  <action name="actionExitGame">
   <property name="text">
    <string>退出游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLowLevel">
   <property name="text">
    <string>初级</string>
   </property>
  </action>
  <action name="actionMidLevel">
   <property name="text">
    <string>中级</string>
   </property>
  </action>
  <action name="actionHighLevel">
   <property name="text">
    <string>高级</string>
   </property>
  </action>
  <action name="actionCustomLevel">
   <property name="text">
    <string>自定义</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>扫雷专业版</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
